server/src/entity/Reservation.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  BaseEntity,
+  Column,
+  CreateDateColumn,
+  Entity,
+  PrimaryGeneratedColumn,
+  ManyToMany,
+  ManyToOne,
+} from 'typeorm';
+import { Restaurant } from './Restaurant';
+import { Account } from './Account';
+
+@Entity()
+export class Reservation extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  public id: string;
+
+  @CreateDateColumn()
+  public dateJoined: Date;
+
+  @Column()
+  public startTime: Date;
+
+  @Column({ default: 0 })
+  public numGuests: number;
+
+  @Column({
+    default: true,
+  })
+  public isActive: boolean;
```

Comment:
why do we need if the reservation is active?

Reply:
canceled maybe ? uh I will remove it. Done